lib/client.js

```diff
@@ -145,6 +146,40 @@ Client.prototype = {
     this._messageHandlers[name].forEach(function(handler) {
       handler(data);
     });
+  },
+
+  /// #### client.request(url, [options], callback)
+  ///
+  /// Dispatch [requests](./requests) via the Zendesk app.
+  ///
+  /// ##### Arguments
+  ///
+  ///   * `url` the url of the request
+  ///   * `options` (optional) an options object that's compatible with `jQuery.ajax`
+  ///   * `callback` a callback that will get called when the request completes
+  ///
+  /// ```javascript
+  /// var client = ZAFClient.init();
+  ///
+  /// app.request('/api/v2/tickets.json', function(tickets) {
+  ///   console.log(tickets);
+  /// });
+  /// ```
+  request: function(url, options, callback) {
+    var uuid = UUID.generate();
```

Comment:
Because there's a risk that someone at some point will write some code that accidentally resets or otherwise messes with requestCount. And it doesn't even have to be in this library — it's a global variable accessible from any JS running in the window.